src/components/Good.js

```diff
@@ -2,10 +2,11 @@ import React from 'react';
 import classes from '../components/Good.module.css';
 
 export function Good(props) {
```

Comment:
Perhaps `Guideline` can take an array of `images` which has the following structure:

`{ src: "url-to-image.svg", alt: "alt text" }`

Then it'd look something like this:

```jsx
<Guideline title="Something" images={{ good: [ { src: "url-to-image.svg", alt: "alt text" } ], bad: [ { src: "url-to-image.svg", alt: "alt text" }, { src: "url-to-image.svg", alt: "alt text" } ] }}
```

Then you can map through these and render them in the component.